cmd/tailscale/cli/status.go

```diff
@@ -212,9 +214,8 @@ func runStatus(ctx context.Context, args []string) error {
 			if ps.ShareeNode {
 				continue
 			}
-			if ps.Location != nil && ps.ExitNodeOption && !ps.ExitNode {
-				// Location based exit nodes are only shown with the
-				// `exit-node list` command.
+			if ps.ExitNodeOption && !ps.ExitNode && strings.HasSuffix(ps.DNSName, mullvadTCD) {
+				// Mullvad exit nodes are only shown with the `exit-node list` command.
 				locBasedExitNode = true
 				continue
 			}
```

Comment:
:question: Non-blocking: Are we planning to provide a way to list all exit nodes?

Reply:
Oh! I misunderstood the comment to mean that `tailscale exit-node list` would _only_ show Mullvad exit nodes. But this comment actually means that `tailscale status` _omits them._ 😂